documentation/index.md

```diff
@@ -342,9 +362,10 @@ exec_ = pipeline.executor(target_nodes=["print_xn"])
 assert ("large data 1", None) == exec_()
 ```
 
-### Debug `ExecNode`
+### **Debug `ExecNode`**
 
-You can Make Debug `ExecNode`s that will only run if `RUN_DEBUG_NODES` env variable is set. These can be visualization `ExecNode`s for example or some complicated Assertions that helps you debug problems when needed that are hostile to the production environment (they consume too much computation time):
+You can make Debug an `ExecNode` that will only run if `RUN_DEBUG_NODES` env variable is set. These can be visualization `ExecNode` 
```

Comment:
```suggestion
You can make Debug an `ExecNode` that will only run if `RUN_DEBUG_NODES` env variable is set. This can be visualization `ExecNode` 
```